### PR TITLE
Small revisions to positioning method names and javadoc

### PR DESCRIPTION
**Pull request by blair1618**

### CODE REVIEW

File: android/tangram/src/com/mapzen/tangram/MapController.java [hunk 1]
```diff
@@ -133,21 +133,21 @@ public void setPosition(LngLat position) {
...
      */
-    public void setPosition(LngLat position, int duration) {
-        setPosition(position, duration, DEFAULT_EASE_TYPE);
+    public void setPositionEased(LngLat position, int duration) {
+        setPositionEased(position, duration, DEFAULT_EASE_TYPE);
     }
 
     /**
-     * Set the geographic position of the center of the map view
+     * Set the geographic position of the center of the map view with easing
```
Comment (tallytalwar) [inline android/tangram/src/com/mapzen/tangram/MapController.java]: Doc should reflect the setting of EaseType in the description? Can you update please?

Comment (blair1618) [inline android/tangram/src/com/mapzen/tangram/MapController.java]: You mean the description should say "cubic easing"? This method doesn't care what easing type is used, if someone does care then they should use the other method. Plus now the docs will remain correct if we change the default easing.

Comment (tallytalwar) [inline android/tangram/src/com/mapzen/tangram/MapController.java]: I just mean to say the description should point the difference between the usage of:

`public void setPositionEased(LngLat position, int duration)` and
`public void setPositionEased(LngLat position, int duration, EaseType ease)`

Comment (blair1618) [inline android/tangram/src/com/mapzen/tangram/MapController.java]: Oh gotcha, will edit that!

Comment (blair1618) [inline android/tangram/src/com/mapzen/tangram/MapController.java]: Updated

**Comment by tallytalwar** [comment]
Looks Good.Merging when travis goes green.

**Pull request merged**